<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tagSelectDialog</class>
 <widget class="QDialog" name="tagSelectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Tag Key</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Description</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>VR</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelValue">
       <property name="text">
        <string>Value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Insert value...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="placeholderText">
        <string>Find text...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOK</sender>
   <signal>clicked()</signal>
   <receiver>tagSelectDialog</receiver>
   <slot>okPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>561</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>tagSelectDialog</receiver>
   <slot>cancelPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>574</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>290</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSearch</sender>
   <signal>textChanged(QString)</signal>
   <receiver>tagSelectDialog</receiver>
   <slot>findText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>okPressed()</slot>
  <slot>cancelPressed()</slot>
  <slot>findText()</slot>
 </slots>
</ui>
